config/routes.rb

```diff
@@ -4,6 +4,10 @@
 
   resources :articles
 
+  resources :rankings, only: [:index]
+
+  resources :statistics, only: [:index]
+
```

Comment:
:+1: for using `only: ...` to only expose the available routes on the new controllers. Good code.